toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -138,6 +140,19 @@ func (s *SystemConfig) IsValid() (err error) {
 		return fmt.Errorf("invalid [KernelCommandLine]: %w", err)
 	}
 
+	// Validate that PackageRepos do not contain duplicate package repo name
+	repoNames := make(map[string]bool)
+	for _, packageRepo := range s.PackageRepos {
+		if err = packageRepo.IsValid(); err != nil {
+			return fmt.Errorf("invalid [PackageRepo]: %w", err)
```

Comment:
Let's also mention the name of the package repo here, to make it easier to find the invalid one.